bin/resetDb_dev.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# This is for development / testing environment.
+
+# Note, to run it, replace:
+#   XXX with your actual mysql user name
+#   YYY with your actual musql password
+
+# Of course, feel free to use host/port and credentials on your local copy of mysql
+
+# Use with caution, as this will completely wipe out and recreate the database
+# metaServ_core in the database server described in the ~/.lsst/dbAuth-metaServ.txt
+mysql --defaults-file=~/.lsst/dbAuth-dbServ.txt -e "drop database XXX_metaServ_core"
+mysql --defaults-file=~/.lsst/dbAuth-dbServ.txt -e "create database XXX_metaServ_core"
+mysql --defaults-file=~/.lsst/dbAuth-metaServ.txt < sql/global.sql
+mysql --defaults-file=~/.lsst/dbAuth-metaServ.txt < sql/dbRepo.sql
+mysql --defaults-file=~/.lsst/dbAuth-metaServ.txt < sql/fileRepo.sql
```

Comment:
Why there are two defaults files here. If they are only different in database name then should we need both of them? Can you use `dbAuth-metaServ.txt` for all commands?